tests/aws/services/lambda_/test_lambda_performance.py

```diff
@@ -352,7 +352,7 @@ def test_lambda_event_source_mapping_sqs(
 ):
     """Test SQS => Lambda event source mapping with concurrent event invokes and validate the number of invocations."""
     # TODO: define IAM permissions
-    num_invocations = 2000
+    num_invocations = 4
```

Comment:
accidental commit?
The latest test are still in the this PR: https://github.com/localstack/localstack/pull/10032